ods_ci/tests/Resources/OCP.resource

```diff
@@ -102,7 +102,15 @@ Wait For Pods To Be Ready
     ...    status_only=${FALSE}
     ${rc}    ${out}=    Run And Return Rc And Output
     ...    oc wait --for=condition=ready pod -l ${label_selector} -n ${namespace} --timeout=${timeout}
-    Should Be Equal As Numbers    ${rc}    ${0}
+    ${pass}=    Run Keyword And Return Status    Should Be Equal As Numbers    ${rc}    ${0}
```

Comment:
why we need this can't we compare the return status directly in below if condition instead of using the above line ?

Reply:
ops, you're right. Dummy me xD

Reply:
done, ty

Reply:
ty